src/community/sldService/pom.xml

```diff
@@ -16,85 +16,103 @@
   
  <groupId>org.geoserver.community</groupId>
  <artifactId>gs-sldservice</artifactId>
- <packaging>jar</packaging>
  <name>REST SLD service</name>
 
- <dependencies>
-  <dependency>
-   <groupId>org.geoserver</groupId>
-   <artifactId>gs-main</artifactId>
-   <version>2.13-SNAPSHOT</version>
-  </dependency>
-  <dependency>
-   <groupId>org.geoserver</groupId>
-   <artifactId>gs-rest</artifactId>
-   <version>2.13-SNAPSHOT</version>
-  </dependency>
-  <dependency>
-   <groupId>org.geoserver</groupId>
-   <artifactId>gs-restconfig</artifactId>
-   <version>2.13-SNAPSHOT</version>
-  </dependency>
-  <dependency>
-   <groupId>org.geoserver</groupId>
-   <artifactId>gs-wms</artifactId>
-   <version>2.13-SNAPSHOT</version>
-  </dependency>
-  <dependency>
-    <groupId>jdom</groupId>
-    <artifactId>jdom</artifactId>
-    <version>1.0</version>
-    <exclusions>
-      <exclusion>
-        <groupId>xerces</groupId>
-        <artifactId>xerces</artifactId>
-      </exclusion>
-    </exclusions>
-  </dependency>
-  <dependency>
-    <groupId>xom</groupId>
-    <artifactId>xom</artifactId>
-    <version>1.1</version>
-  </dependency>
-  <dependency>
-   <groupId>org.geoserver</groupId>
-   <artifactId>gs-main</artifactId>
-   <version>${project.version}</version>
-   <classifier>tests</classifier>
-   <scope>test</scope>
-  </dependency>
-  <dependency>
-   <groupId>org.geoserver</groupId>
-   <artifactId>gs-wcs1_0</artifactId>
-   <version>${project.version}</version>
-   <scope>test</scope>
-  </dependency>
-  <dependency>
-   <groupId>org.geoserver</groupId>
-   <artifactId>gs-wcs1_1</artifactId>
-   <version>${project.version}</version>
-   <scope>test</scope>
-  </dependency>
-  <dependency>
-   <groupId>org.geoserver.extension</groupId>
-   <artifactId>gs-h2</artifactId>
-   <version>${project.version}</version>
-   <scope>test</scope>
-  </dependency>
-  <dependency>
-   <groupId>org.springframework</groupId>
-   <artifactId>spring-test</artifactId>
-   <scope>test</scope>
-  </dependency>
-  <dependency>
-   <groupId>xmlunit</groupId>
-   <artifactId>xmlunit</artifactId>
-   <scope>test</scope>
-  </dependency>
-  <dependency>
-     <groupId>org.mockito</groupId>
-     <artifactId>mockito-core</artifactId>
-     <scope>test</scope>
-  </dependency>    
- </dependencies>
+	<dependencies>
```

Comment:
Mass reformat, using tabs instead of spaces.

Reply:
Done

Reply:
And here too, still using tabs